com.simplenote.Simplenote.desktop

```diff
@@ -1,6 +1,6 @@
 [Desktop Entry]
 Name=Simplenote
-Exec=simplenote %U
+Exec=simplenote --enable-features=UseOzonePlatform --ozone-platform-hint=auto  --enable-wayland-ime --wayland-text-input-version=3 %U
```

Comment:
It's probably better to put these arguments in a wrapper script that is executed when running the flatpak. Like that, the arguments are also used when simplenote is started via `flatpak run` on the command line, not only when started via the `.desktop` file.